helm_deploy/values-prod.yaml

```diff
@@ -12,6 +12,7 @@ generic-service:
   env:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
     API_BASE_URL_OAUTH: https://sign-in.hmpps.service.justice.gov.uk/auth
+    FEATURE_RECORD-LOGON-DATE: true
```

Comment:
will remove the feature switch in a later PR once we are confident is all working ok